src/SprykerEco/Zed/AfterPay/Business/Payment/Mapper/OrderToRequestTransfer.php

```diff
@@ -168,6 +170,31 @@ protected function buildOrderWithItemsRequestTransfer(AfterPayCallTransfer $afte
 
         $orderRequestTransfer = $this->addExpensesToOrderRequestTransfer($orderRequestTransfer, $afterPayCallTransfer);
         $orderRequestTransfer = $this->addGiftcardItems($orderRequestTransfer, $afterPayCallTransfer);
+        $orderRequestTransfer = $this->addShippingFeeToOrderRequestTransfer($orderRequestTransfer, $afterPayCallTransfer);
+
+        return $orderRequestTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\AfterPayRequestOrderTransfer $orderRequestTransfer
+     * @param \Generated\Shared\Transfer\AfterPayCallTransfer $afterPayCallTransfer
+     *
+     * @return \Generated\Shared\Transfer\AfterPayRequestOrderTransfer
+     */
+    protected function addShippingFeeToOrderRequestTransfer(
+        AfterPayRequestOrderTransfer $orderRequestTransfer,
+        AfterPayCallTransfer $afterPayCallTransfer
+    ): AfterPayRequestOrderTransfer {
+        $amount = (string)$this->moneyFacade
+            ->convertIntegerToDecimal($afterPayCallTransfer->getTotals()->getShipmentTotal());
+
+        $orderRequestTransfer->addItem(
+            (new AfterPayRequestOrderItemTransfer())
+                ->setProductId(static::SHIPPING_FEE_PROVIDER)
```

Comment:
is it correct that ProductId consist a string?

Reply:
I think it's correct. It's special item withut productId like as GiftCardItem:
`line 348 ->setProductId(static::GIFT_CARD_PROVIDER . $index)`